lib/src/components/list_item/list_item.dart

```diff
@@ -2,205 +2,224 @@ import 'package:flutter/foundation.dart';
+    required this.primaryText,
+    required dynamic value,
+    this.secondaryText,
+    this.leading,
+    this.showDivider,
+    dynamic groupValue,
+    super.key,
+    ValueChanged<dynamic>? onChanged,
+  })  : trailing = ZetaRadio<dynamic>(
+          value: value,
+          groupValue: groupValue,
+          onChanged: onChanged,
+        ),
+        onTap = (() => onChanged?.call(value));
 
   /// A Widget to display before the title;
   final Widget? leading;
 
   /// Called when user taps on the [ZetaListItem].
   final VoidCallback? onTap;
 
-  /// Applies selected styles. If selected is true trailing mu
-  final bool selected;
-
-  /// Additional content displayed over the title.
-  /// Typically a [Text] widget.
-  final Widget? subtitle;
+  /// The primary text of the [ZetaListItem].
+  final String primaryText;
 
-  /// The primary content of the [ZetaListItem].
-  final Widget title;
+  /// The primary text of the [ZetaListItem].
+  final String? secondaryText;
 
   /// A widget to display after the title.
   final Widget? trailing;
 
+  /// Adds a border to the bottom of the list item.
+  final bool? showDivider;
+
   @override
   void debugFillProperties(DiagnosticPropertiesBuilder properties) {
     super.debugFillProperties(properties);
     properties
-      ..add(DiagnosticsProperty<bool>('dense', dense))
-      ..add(DiagnosticsProperty<bool>('enabled', enabled))
-      ..add(DiagnosticsProperty<bool>('enabledDivider', enabledDivider))
       ..add(DiagnosticsProperty<Widget>('leading', leading))
       ..add(ObjectFlagProperty<VoidCallback?>.has('onTap', onTap))
-      ..add(DiagnosticsProperty<bool>('selected', selected))
-      ..add(DiagnosticsProperty<Widget>('subtitle', subtitle))
-      ..add(DiagnosticsProperty<Widget>('title', title))
-      ..add(DiagnosticsProperty<Widget>('trailing', trailing));
+      ..add(DiagnosticsProperty<Widget>('trailing', trailing))
+      ..add(StringProperty('label', primaryText))
+      ..add(StringProperty('secondaryText', secondaryText))
+      ..add(DiagnosticsProperty<bool?>('showDivider', showDivider));
   }
 
-  TextStyle get _titleTextStyle => dense ? ZetaTextStyles.titleSmall : ZetaTextStyles.titleMedium;
-
   @override
   Widget build(BuildContext context) {
-    final zetaColors = Zeta.of(context).colors;
-
-    return _ListItemContainer(
-      enabled: enabled,
-      selected: selected,
-      onTap: onTap,
-      dense: dense,
-      enabledDivider: enabledDivider,
-      child: Row(
-        mainAxisAlignment: MainAxisAlignment.spaceBetween,
-        children: [
-          Flexible(
+    final colors = Zeta.of(context).colors;
+    final divide = showDivider ?? _DivderScope.of(context)?.showDivider ?? false;
+
+    return Container(
+      constraints: const BoxConstraints(minHeight: ZetaSpacing.xl_9),
+      decoration: BoxDecoration(
+        border: Border(
+          bottom: BorderSide(
+            color: divide ? colors.borderDefault : Colors.transparent,
```

Comment:
does having a bottom border affect the height of the item?
Looks like item is 63 high regardless of divider? but that seems wrong to me
![image](https://github.com/ZebraDevs/zeta_flutter/assets/23633655/27663669-38b7-4734-ab54-302863461b5d)

Reply:
Yes it does, I noticed in widget book when I toggled the border it shifted the interior elements. I thought this was a decent work around so the leading/trailing didn't get misaligned